exercises/00-getting-started.md

```diff
@@ -28,32 +28,33 @@ exercise.
 A general overview of the exercise is given first, followed by more detailed step-by-step instructions.
 If you want a little challenge, you can try to just read the general steps of the exercise and if you
 need more help, take a look at the step-by-step instructions. The step-by-step instructions also
-explain some steps about _why_ they are done as they are done in the exercise.
+include explanations in some places about _why_ things are done as they are done in the exercise.
 
 The step-by-step instructions are divided into SeleniumLibrary and Browser sections separately.
 
 **Each exercise is summarized in bold text at the beginning.**
 
 All exercises have some explanation on why some things are done as
-they are in normal paragrahs.
+they are in normal paragraphs.
 
-- All exercise steps are bulleted. The bulleted points summarize the previous
-paragraphs as small individual tasks. **To finish the exercise, it's sufficient to read
-only the bulleted steps.** All other text is just narrative and explanations as to why things are
-done as they are.
+- All exercise steps are in bullet points. Each bullet points is an individual small task summary.
+Between the bullet points there are longer descriptive paragraphs.
+ **To complete the exercise, it's ok to skip those paragraphs.**
+Read the paragraphs for more explanations and the solution narrative, so you can understand and apply it in your work.
```

Comment:
I do not understand what "solution narrative" means in this sentence. Maybe just:
```suggestion
Read the paragraphs for more explanation, so you can understand and apply it in your work.
```
?

Reply:
Since there is no katas here to check results, in order to help people progress but to avoid giving answers directly, there is a narrative that supports every exercise guiding participants through logic of how the exercises could be solved.